app/services/ab_test_service.rb

```diff
@@ -0,0 +1,23 @@
+# This module wraps the FieldTest to provide a bit of insulation
+# between the FieldTest implementation and the application logic.
+module AbTestService
```

Comment:
This is a bit of a deviation from our normal naming scheme. We don't usually add a `Service` suffix. Also, in general, we follow a naming scheme of models -> nouns (DB-backed or not DB-backed), services -> verbs. That said, we do have some exceptions to the rule, e.g. `app/services/feature_flag`.

Reply:
I went ahead and renamed `app/models/ab_experiment.rb`. I like moving away from test and towards experiment so when we talk about the object we're not conflating the module with our source code testing systems.